core/chains/evm/logpoller/log_poller_test.go

```diff
@@ -105,52 +106,54 @@ func BenchmarkSelectLogsCreatedAfter(b *testing.B) {
 func TestPopulateLoadedDB(t *testing.T) {
 	t.Skip("Only for local load testing and query analysis")
 	_, db := heavyweight.FullTestDBV2(t, nil)
+	ctx := testutils.Context(t)
 	chainID := big.NewInt(137)
 
-	o := logpoller.NewORM(big.NewInt(137), db, logger.Test(t), pgtest.NewQConfig(true))
+	o := logpoller.NewORM(big.NewInt(137), db, logger.Test(t))
 	event1, address1, address2 := populateDatabase(t, o, chainID)
 
 	func() {
 		defer logRuntime(t, time.Now())
-		_, err1 := o.SelectLogs(750000, 800000, address1, event1)
+		_, err1 := o.SelectLogs(ctx, 750000, 800000, address1, event1)
 		require.NoError(t, err1)
 	}()
 	func() {
 		defer logRuntime(t, time.Now())
-		_, err1 := o.SelectLatestLogEventSigsAddrsWithConfs(0, []common.Address{address1}, []common.Hash{event1}, 0)
+		_, err1 := o.SelectLatestLogEventSigsAddrsWithConfs(ctx, 0, []common.Address{address1}, []common.Hash{event1}, 0)
 		require.NoError(t, err1)
 	}()
 
 	// Confirm all the logs.
-	require.NoError(t, o.InsertBlock(common.HexToHash("0x10"), 1000000, time.Now(), 0))
+	require.NoError(t, o.InsertBlock(ctx, common.HexToHash("0x10"), 1000000, time.Now(), 0))
 	func() {
 		defer logRuntime(t, time.Now())
-		lgs, err1 := o.SelectLogsDataWordRange(address1, event1, 0, logpoller.EvmWord(500000), logpoller.EvmWord(500020), 0)
+		lgs, err1 := o.SelectLogsDataWordRange(ctx, address1, event1, 0, logpoller.EvmWord(50000), logpoller.EvmWord(50020), 0)
```

Comment:
Why did these values need to change? Since there shouldn't be a behavior change in the query, worried this means something did.

Reply:
This test was outdated. It was created for local testing and probably hasn't been run by anyone. When I ran it on develop it also needed to be updated to pass which is why I made these changes here.